applications/infra.py

```diff
@@ -0,0 +1,160 @@
+import matplotlib.pyplot as plt
```

Comment:
what is the aim of this file?

Reply:
It is an example for people who want something simple... without having to understand the machinery in experiments. Here you just run "python infra.py" and it works.

Reply:
Then maybe you want to refer to it somewhere in the docs, and possibly even move it to the docs folder actually, because in the end that's what it is isnt it?